src/routes/Settings.tsx

```diff
@@ -100,13 +118,119 @@ const Settings: React.FC = () => {
+        key: 'pupil',
+        render: (certificate: IExposedCertificate) =>
+          `${certificate.pupil.firstname} ${certificate.pupil.lastname}`,
+      },
+      {
+        title: 'Student*in',
+        key: 'student',
+        render: (certificate: IExposedCertificate) =>
+          `${certificate.student.firstname} ${certificate.student.lastname}`,
+      },
+      {
+        title: 'Fächer',
+        key: 'subjects',
+        render: (certificate: IExposedCertificate) => (
+          <>
+            {certificate.subjects.split(',').map((subject) => (
+              <Tag key={subject}>{subject.toUpperCase()}</Tag>
+            ))}
+          </>
+        ),
+      },
+      {
+        title: 'Status',
+        key: 'status',
+        filters: ['manual', 'awaiting-approval', 'approved'].map((s) => ({
+          value: s,
+          text: stateTranslation[s],
+        })),
+        filterMultiple: false,
+        onFilter: (value: string, certificate: IExposedCertificate) =>
+          certificate.state === value,
+        // defaultSortOrder: 'descend',
+        sorter: (a: IExposedCertificate, b: IExposedCertificate) =>
+          a.state.length - b.state.length,
+        // sortDirections: ['descend', 'ascend'],
+        render: (certificate: IExposedCertificate) => (
+          <>
+            <Tag
+              color={
+                (certificate.state === 'manual' && 'green') ||
+                (certificate.state === 'awaiting-approval' && 'yellow') ||
+                (certificate.state === 'approved' && 'green')
+              }
+              key={certificate.state}
+            >
+              {stateTranslation[certificate.state]}
+            </Tag>
+          </>
+        ),
+      },
+      {
+        title: 'Aktion',
+        key: 'action',
+        render: (certificate: IExposedCertificate) => (
+          <Space size="middle">
+            {userContext.user.isTutor &&
+              certificate.state !== 'awaiting-approval' && (
+                <>
+                  <Select
+                    defaultValue={defaultLanguage}
+                    onChange={(event) => setLanguage(event)}
+                    style={{ width: 120 }}
+                  >
+                    {Object.entries(supportedLanguages).map(([code, value]) => (
+                      <Select.Option value={code}>{value}</Select.Option>
+                    ))}
+                  </Select>
+                  <Button
+                    type="primary"
+                    onClick={() => showCertificate(certificate.uuid)}
+                  >
+                    Ansehen
+                  </Button>
+                </>
+              )}
+
+            {/* <Button danger>Löschen</Button> */}
+          </Space>
+        ),
+      },
+    ];
+
+    return (
+      <>
+        <Title size="h3" className={classes.subjectTitle}>
+          Zertifikate
+        </Title>
+        <Table dataSource={certificates.value ?? []} columns={columns} />
```

Comment:
Maybe you can add a horizontal scrollbar for the table to make it displayable on smaller screens, especially on mobile screens.

Reply:
created a followup ticket on our [project board](https://github.com/corona-school/project-user/projects/5).